dracut/30ignition/ignition-generator

```diff
@@ -39,6 +39,10 @@ add_requires() {
 if $(cmdline_bool 'ignition.firstboot' 0); then
     add_requires ignition-complete.target initrd.target
 
+    # For consistency tear down the network between the initramfs and
+    # real root. See https://github.com/coreos/fedora-coreos-tracker/issues/394#issuecomment-599721763
+    add_requires coreos-teardown-initramfs-network.service ignition-complete.target
```

Comment:
We don't have to do this from the generator, right? We can just statically hook it to `ignition-complete.target` like the other services.

Reply:
correct. i forward ported this from spec2x. I've modified it to be statically enabled.